ApiTestCase.php

```diff
@@ -0,0 +1,260 @@
+<?php
+
+namespace AppBundle\Test;
+
+use Coduo\PHPMatcher\Factory\SimpleFactory;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase as BaseWebTestCase;
+use Symfony\Component\Finder\Finder;
+use Doctrine\Common\DataFixtures\Purger\ORMPurger;
+use Nelmio\Alice\Loader\Yaml as Loader;
+use Symfony\Component\HttpFoundation\Response;
+use Doctrine\ORM\EntityManagerInterface;
+
+abstract class ApiTestCase extends BaseWebTestCase
+{
+    /**
+     * @var \Symfony\Bundle\FrameworkBundle\Client $client
+     */
+    protected $client;
+
+    /**
+     * @var EntityManagerInterface
+     */
+    protected $defaultEntityManager;
+
+    public function __construct()
+    {
+        $this->defaultEntityManager = $this->get('doctrine.orm.default_entity_manager');
```

Comment:
do we need to know that this is the default one? if some would need another one he/she could get it from container anyway